plugins/texteditor.koplugin/main.lua

```diff
@@ -66,6 +67,7 @@ function TextEditor:loadSettings()
     self.auto_para_direction = self.settings:nilOrTrue("auto_para_direction")
     self.force_ltr_para_direction = self.settings:isTrue("force_ltr_para_direction")
     self.qr_code_export = self.settings:nilOrTrue("qr_code_export")
+    self.show_kbd_on_start = self.settings:nilOrTrue("show_kbd_on_start")
```

Comment:
No real need to abbreviate it, `show_keyboard_on_start` is not much longer, and reads nicer :)